controllers/surveyController.js

```diff
@@ -2,7 +2,7 @@ const db = require("../models");
 
 module.exports = {
   findAllSurveys: () => db.Survey.find({}),
-
+  
```

Comment:
what's with the extra space here?

Reply:
An accident I guess, it just an extra space